src/bepasty/views/display.py

```diff
@@ -110,6 +110,10 @@ def read_data(item):
                     src = url_for('bepasty.inline', name=name)
                     link_txt = u'Click to see PDF'
                     rendered_content = Markup(u'<a href="%s">%s</a>' % (src, link_txt))
+                elif ct == 'application/x-asciinema-recording':
+                    src = url_for('bepasty.download', name=name)
+                    rendered_content = Markup(u'<asciinema-player src="%s" title="%s" theme="%s" autoplay>' %
+                                              (src, item.meta[FILENAME], current_app.config['ASCIINEMA_THEME']))
```

Comment:
can this be used for XSS by using strange filenames?

Reply:
I doubt it. In my tests, it used the generated filename: `/LdigCtA9/+download`

Reply:
Looks like bepasty handles that. I uploaded `demo.cast" lal.cast` which resulted in:
```
<asciinema-player src="/hFFgG6eb/+download" title="demo.cast lal.cast" theme="asciinema" autoplay="">
```

Reply:
hmm, ok, so it seems the filename given by the user gets preprocessed somewhere else.

Reply:
guess this should have a theme default like:
```
current_app.config.get('ASCIINEMA_THEME', 'asciinema')
```

Reply:
bepasty preprocesses the filename before storing it into metadata and uses this regex (all matching chars will be stripped from filename):

```
class Upload(object):
    _filename_re = re.compile(r'[^a-zA-Z0-9 \*+:;.,_-]+')
```